moderncvheadiii.sty

```diff
@@ -85,7 +85,7 @@
     \ifthenelse{\equal{\@title}{}}{}{\titlestyle{~|~\@title}}% \isundefined doesn't work on \@title, as LaTeX itself defines \@title (before it possibly gets redefined by \title) 
     % optional detailed information
     \if@details{%
-      \\%
+      %\\%
```

Comment:
Shouldn't this line then just be removed?

Reply:
Yes. I just had it like that to remind me that it used to be there.